OpenRA.Mods.Common/Traits/Player/ProductionQueue.cs

```diff
@@ -111,6 +111,8 @@ public ProductionQueue(ActorInitializer init, Actor playerActor, ProductionQueue
 			CacheProducibles(playerActor);
 			allProducibles = producible.Where(a => a.Value.Buildable || a.Value.Visible).Select(a => a.Key);
 			buildableProducibles = producible.Where(a => a.Value.Buildable).Select(a => a.Key);
+			if (info.LowPowerSlowdown <= 0)
+				throw new YamlException("Production queue must have LowPowerSlowdown of at least 1.");
```

Comment:
Please check this in `ProductionQueueInfo` instead (search for `YamlException` to see examples of how).